### PR TITLE
Migrate to SKY UX 3

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: src/app/public/modules/media-query/media-query.service.ts [hunk 1]
```diff
@@ -37,10 +37,10 @@ export class SkyMediaQueryService {
   private mdMql: MediaQueryList;
   private lgMql: MediaQueryList;
 
-  private xsListener: MediaQueryListListener;
-  private smListener: MediaQueryListListener;
-  private mdListener: MediaQueryListListener;
-  private lgListener: MediaQueryListListener;
+  private xsListener: any;
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/media-query/media-query.service.ts]: Why are we changing these? Properties which are `any` in main code classes make me nervous :)

File: src/app/public/modules/numeric/numeric.options.ts [hunk 2]
```diff
@@ -3,6 +3,6 @@ export class NumericOptions {
   public format = 'number';
   public iso = 'USD';
   public minDigits?: number;
-  public truncate? = true;
-  public truncateAfter? = 0;
+  public truncate?: boolean = true;
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/numeric/numeric.options.ts]: I thought at one point you said we didn't need the type declarations if the property was initialized. Why are you adding them here?

File: src/app/public/modules/media-query/media-query.service.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/media-query/media-query.service.ts]: The type `MediaQueryListListener` was removed in TypeScript 3.1 (https://stackoverflow.com/questions/53010349/using-mediaquerylistlistener-in-typescript-3-1), and the type itself isn't well supported (https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryListListener). Ideally, we would refactor this service to work with "approved" types, but that will take some doing.

File: src/app/public/modules/numeric/numeric.options.ts (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/numeric/numeric.options.ts]: You are correct, however, the later version of TypeScript screams at me if we don't provide it (and use the "optional" marker `?`).

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/numeric/numeric.options.ts]: You are correct, however, the later version of TypeScript screams at me if we don't provide the type (and we also wish to use the "optional" marker `?`).

File: src/app/public/modules/media-query/media-query.service.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/media-query/media-query.service.ts]: Ok. Sounds good.

File: src/app/public/modules/numeric/numeric.options.ts (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/numeric/numeric.options.ts]: Ok. Sounds good.

**Pull request merged**